app/views/admin/overview/index.html.erb

```diff
@@ -0,0 +1,54 @@
+<h2 class="crayons-title mb-3">Overview</h2>
+<div class="grid s:grid-cols-3 s:grid-rows-4 gap-3">
+  <article style="grid-column: span 2; grid-row: span 4;">
+    <section class="crayons-card mb-3">
+      <div class="crayons-card__body" data-controller="ahoy">
+        <h3 class="crayons-subtitle-2">Welcome Checklist</h3>
+        <p class="crayons-field__description mb-3">Here are things we recommend you do now that you're on Forem</p>
```

Comment:
```suggestion
        <p class="crayons-field__description mb-3">Here are a few things we recommend doing now that you're using Forem!</p>
```

I know the design mock has different text, but when I was playing with it locally, this just read a little funky to me. What do you think about this instead?